### PR TITLE
Added Form Design Content

### LINKED ISSUE #290: Consider Adding Topic: Form design
See #265

### PR DESCRIPTION
**Pull request by loganfranken**
Fixes #290

### CODE REVIEW

**Comment by loganfranken** [comment]
Hey there! Did this look good? Anything I can add or change?

File: html.md [hunk 1]
```diff
@@ -438,6 +438,127 @@ In this case, you can use the `aria-label` attribute:
...
+  width: 3em;
+}
+
+.name-input
+{
+  width: 10em;
+}
+```
+
+In this example, the styling for the "age" input has a shorter width While
```
Comment (dsaludares) [inline html.md]: "While" is capitalized.

File: html.md [hunk 2]
```diff
@@ -438,6 +438,127 @@ In this case, you can use the `aria-label` attribute:
...
+```html
+<!-- GOOD PRACTICE -->
+<input type="text" name="age" id="age" maxlength="3">
+```
+
+For a more advanced technique, you could also use JavaScript to provide a
+"text counter" or a dynamic count displayed near a field of how many
+characters a user can type before reaching the maximum length.
+
+Overall, when fitting fields to content, consider the user experience: if you
```
Comment (dsaludares) [inline html.md]: Remove "you".

**Review by dsaludares — COMMENTED** [review]
Hi @loganfranken!

Sorry that this took awhile to get to.  Thank you for your work on this!  I have minor changes.  Thanks again!

**Comment by loganfranken** [comment]
No need to be sorry! Thanks for catching those, @dsaludares! Updated!

**Comment by dsaludares** [comment]
Thank you, @loganfranken!

**Pull request merged**